lib/components/ActionTrigger/ActionTrigger.module.scss

```diff
@@ -39,13 +36,6 @@ $line-height: 4.5*$grid-size;
         }
     }
 
-    &:hover:not(.disabled) {

```

Comment:
I didn't see this selector in the styled components code. Is there a way to say hover and not disable in styled components?

Reply:
SC use normal CSS code, so we can use same selector if needed. However it looks like this selector is not actually useful, I will dig more into it.

Reply:
If it's not useful I'm in favor of removing unnecessary code. Maybe @vincenzocaruso has more context on this.

Reply:
Just double checked, seems it's not useful